CHANGELOG.rst

```diff
@@ -1,13 +1,48 @@
 sw_usb_audio Change Log
 =======================
 
-UNRELEASED
-----------
+8.1.0
+-----
 
+  * ADDED:     2AMi18o18mssaax build config (MIDI, SPDIF Rx/Tx & ADAT Rx/Tx)
+  * ADDED:     2AMi16o16xxxaax build config (ADAT Rx/Tx)
+  * ADDED:     2AMi18o8_mxxxxx build config (MIDI)
```

Comment:
Should be 8 inputs instead of 18, and no underscore in the config name

Reply:
Thanks. I'll update spec document too (which also has this mistake)